### ISSUE TITLE
iPhone 5 issue

### ISSUE THREAD

**Issue by hatzipanis** [lukehaas/Scrollify#71]
I have an issue with Scrollify on iPhone 5 (not 5S which behaves fine). While this is a screenshot from BrowserStack's emulator, I'm able to replicate this on a real iPhone 5 device.

You can view the site [here](http://enjoymortgage.wpengine.com/).

I believe this is something to do with the iPhone UI but not 100% sure. The shortened sections occur on all sections that Scrollify acts on. Any ideas? 

![bs_ios_mobile_iphone 5-6 0](https://cloud.githubusercontent.com/assets/1661180/12768505/e1cf0974-ca74-11e5-855d-bea67ba7c74e.jpg)

**Comment by hatzipanis** [comment]
It seems to be the height of the address bar which is displayed then disappears.

**Comment by lukehaas** [comment]
Which version of iOS does your iPhone 5 have?

**Comment by hatzipanis** [comment]
I've been testing on BrowserStack with an iPhone 5 with iOS6. Unfortunately I can't confirm the real device iOS version just now.

**Comment by lukehaas** [comment]
Your project actually has a requirement to support iOS 6?

This is what the usage stats look like for iPhones:
9.X	77.8%
8.X	11.9%
7.X	9.2%
6.X	1.0%
5.X	0.1%
4.X	0.0%

I don't actually have access to a device with iOS 6, so I'm afraid it's not something I can support.

**Comment by hatzipanis** [comment]
You're absolutely right. It's not something I usually check for but a client had one of these devices and asked me about this issue. It wasn't until after I posted it here did I realise how low the usage stats were. I've already communicated to the client that this is not something we'd fix but I thought to see if you knew what the issue was anyway, just to understand. 

Thanks anyway, appreciate the response!
